src/hardware.md

```diff
@@ -37,3 +63,271 @@ We can write import scripts to handle virtually any "open" file format that cont
+
+```xml
+<node name="free_fleet_client_node" 
+    pkg="free_fleet_client_ros1"
+    type="free_fleet_client_ros1" output="screen">
+
+  <!-- These parameters will be used to identify the mobile robots -->
+  <param name="fleet_name" type="string" value="example_fleet"/>
+  <param name="robot_name" type="string" value="example_bot"/>
+  <param name="robot_model" type="string" value="Turtlebot3"/>
+
+  <!-- These are the topics required to get battery and level information -->
+  <param name="battery_state_topic" type="string" value="example_bot/battery_state"/>
+  <param name="level_name_topic" type="string" value="example_bot/level_name"/>
+
+  <!-- These frames will be used to update the mobile robot's location -->
+  <param name="map_frame" type="string" value="example_bot/map"/>
+  <param name="robot_frame" type="string" value="example_bot/base_footprint"/>
+
+  <!-- The name of the move_base server for actions -->
+  <param name="move_base_server_name" type="string" value="example_bot/move_base"/>
+
+  <!-- These are DDS configurations used between Free Fleet clients and servers -->
+  <param name="dds_domain" type="int" value="42"/>
+  <param name="dds_state_topic" type="string" value="robot_state"/>
+  <param name="dds_mode_request_topic" type="string" value="mode_request"/>
+  <param name="dds_path_request_topic" type="string" value="path_request"/>
+  <param name="dds_destination_request_topic" type="string" value="destination_request"/>
+  
+  <!-- This decides how long the client should wait for a valid transform and action server before failing -->
+  <param name="wait_timeout" type="double" value="10"/>
+  
+  <!-- These define the frequency at which the client checks for commands and 
+  publishes the robot state to the server -->
+  <param name="update_frequency" type="double" value="10.0"/>
+  <param name="publish_frequency" type="double" value="1.0"/>
+
+  <!-- The client will only pass on navigation commands if the destination or first waypoint 
+  of the path is within this distance away, otherwise it will ignore the command -->
+  <param name="max_dist_to_first_waypoint" type="double" value="10.0"/>
+
+</node>
+```
+
+The running `free_fleet` client will communicate with the nodes running on the robot via ROS1, while publishing its state and subscribing to requests over DDS with the `free_fleet` Server.
+
+The current implementation of the `free_fleet` server is implemented with ROS2 and communicates with RMF using the aforementioned ROS2 message and topic interfaces of an RMF fleet adapter. The ROS2 build instructions can also be found on the same repository. Similar to the client, a simple ROS2 wrapper has been implemented, and it can be started using a `.launch.xml` file like so,
```

Comment:
```suggestion
The current implementation of the `free_fleet` server is implemented with ROS 2 and communicates with RMF using the aforementioned ROS 2 message and topic interfaces of an RMF fleet adapter. The ROS 2 build instructions can also be found on the same repository. Similar to the client, a simple ROS2 wrapper has been implemented, and it can be started using a `.launch.xml` file like so,
```